### PR TITLE
Created an introductory text for 01_overview.stories.mdx

### PR DESCRIPTION
**Pull request by raja-wmde**
This is a first proposal, feel free to adapt.

### CODE REVIEW

File: docs/src/doc/create-reusable-components/01_overview.stories.mdx [hunk 1]
```diff
@@ -4,4 +4,21 @@ import { Meta } from '@storybook/addon-docs/blocks';
 
 # Overview
 
-high-level overview of the process and roles involved, will redirect readers to the other pages for further instructions
+The following documentation pages will give you detailed descriptions how to create new components for the WiKit library that can be reused by other teams as well. Before creating a new component, please check the existing components and consider whether you rather want to [Modify an existing components](?path=/story/documentation-how-to-modify-existing-components--page#how-to-modify-existing-components) (the page gives an overview in which cases you might want to modify a component).
```
Comment (SaiSan-WMDE) [inline docs/src/doc/create-reusable-components/01_overview.stories.mdx:7]:
```suggestion
The following documentation pages will give you detailed descriptions on how to create new components for the WiKit library that can be reused by other teams as well. Before creating a new component, please check the existing components and consider whether you rather want to [Modify an existing component](?path=/story/documentation-how-to-modify-existing-components--page#how-to-modify-existing-components) (the page gives an overview of the cases in which you might want to modify a component). 
```

File: docs/src/doc/create-reusable-components/01_overview.stories.mdx [hunk 2]
```diff
@@ -4,4 +4,21 @@ import { Meta } from '@storybook/addon-docs/blocks';
 
 # Overview
 
-high-level overview of the process and roles involved, will redirect readers to the other pages for further instructions
+The following documentation pages will give you detailed descriptions how to create new components for the WiKit library that can be reused by other teams as well. Before creating a new component, please check the existing components and consider whether you rather want to [Modify an existing components](?path=/story/documentation-how-to-modify-existing-components--page#how-to-modify-existing-components) (the page gives an overview in which cases you might want to modify a component). 
+
+For creating a new component, we see two main steps: [Designing Components](?path=/story/documentation-how-to-create-reusable-components-designing-components--page#1-designing-new-system-components) with designers as the target group for the documentation and [Developing Components](?path=/story/documentation-how-to-create-reusable-components-developing-components--page#developing-components) with developers as the target group for the documentation.
```
Comment (SaiSan-WMDE) [inline docs/src/doc/create-reusable-components/01_overview.stories.mdx:9]:
```suggestion
For creating a new component, we see two main steps: [Designing Components](?path=/story/documentation-how-to-create-reusable-components-designing-components--page#1-designing-new-system-components), with designers as the target group for the documentation, and [Developing Components](?path=/story/documentation-how-to-create-reusable-components-developing-components--page#developing-components), with developers as the target group for the documentation.
```

File: docs/src/doc/create-reusable-components/01_overview.stories.mdx [hunk 3]
```diff
@@ -4,4 +4,21 @@ import { Meta } from '@storybook/addon-docs/blocks';
...
+
+
+
+**The Designing Components page covers the following topics:**
+
+- [designing new system components](?path=/story/documentation-how-to-create-reusable-components-designing-components--page#1-designing-new-system-components) (including the [design definition of done](?path=/story/documentation-how-to-create-reusable-components-designing-components--page#15-the-design-definition-of-done))
```
Comment (SaiSan-WMDE) [inline docs/src/doc/create-reusable-components/01_overview.stories.mdx:15]:
```suggestion
- [Designing new system components](?path=/story/documentation-how-to-create-reusable-components-designing-components--page#1-designing-new-system-components) (including the [design definition of done](?path=/story/documentation-how-to-create-reusable-components-designing-components--page#15-the-design-definition-of-done))
```

File: docs/src/doc/create-reusable-components/01_overview.stories.mdx [hunk 4]
```diff
@@ -4,4 +4,21 @@ import { Meta } from '@storybook/addon-docs/blocks';
...
+- [using design tokens](?path=/story/documentation-how-to-create-reusable-components-designing-components--page#2-using-design-tokens)
```
Comment (SaiSan-WMDE) [inline docs/src/doc/create-reusable-components/01_overview.stories.mdx:16]:
```suggestion
- [Using design tokens](?path=/story/documentation-how-to-create-reusable-components-designing-components--page#2-using-design-tokens)
```

File: docs/src/doc/create-reusable-components/01_overview.stories.mdx [hunk 5]
```diff
@@ -4,4 +4,21 @@ import { Meta } from '@storybook/addon-docs/blocks';
...
+- [design handover to developers](?path=/story/documentation-how-to-create-reusable-components-designing-components--page#3-design-handover)
```
Comment (SaiSan-WMDE) [inline docs/src/doc/create-reusable-components/01_overview.stories.mdx:17]:
```suggestion
- [Design handover to developers](?path=/story/documentation-how-to-create-reusable-components-designing-components--page#3-design-handover)
```

File: docs/src/doc/create-reusable-components/01_overview.stories.mdx [hunk 6]
```diff
@@ -4,4 +4,21 @@ import { Meta } from '@storybook/addon-docs/blocks';
...
+
+
+**The Developing Components page covers the following topics:**
+
+- [working with component level design tokens](?path=/story/documentation-how-to-create-reusable-components-developing-components--page#working-with-component-level-design-tokens)
```
Comment (SaiSan-WMDE) [inline docs/src/doc/create-reusable-components/01_overview.stories.mdx:22]:
```suggestion
- [Working with component level design tokens](?path=/story/documentation-how-to-create-reusable-components-developing-components--page#working-with-component-level-design-tokens)
```

Comment (SaiSan-WMDE) [inline docs/src/doc/create-reusable-components/01_overview.stories.mdx:22]:
```suggestion
- [Working with component-level design tokens](?path=/story/documentation-how-to-create-reusable-components-developing-components--page#working-with-component-level-design-tokens)
```

File: docs/src/doc/create-reusable-components/01_overview.stories.mdx [hunk 7]
```diff
@@ -4,4 +4,21 @@ import { Meta } from '@storybook/addon-docs/blocks';
...
+- [releasing a new component](?path=/story/documentation-how-to-create-reusable-components-developing-components--page#releasing-a-new-component)
```
Comment (SaiSan-WMDE) [inline docs/src/doc/create-reusable-components/01_overview.stories.mdx:23]:
```suggestion
- [Releasing a new component](?path=/story/documentation-how-to-create-reusable-components-developing-components--page#releasing-a-new-component)
```

File: docs/src/doc/create-reusable-components/01_overview.stories.mdx [hunk 8]
```diff
@@ -4,4 +4,21 @@ import { Meta } from '@storybook/addon-docs/blocks';
...
+- [Testing Components](?path=/story/documentation-how-to-create-reusable-components-developing-components--page#testing-components)
```
Comment (SaiSan-WMDE) [inline docs/src/doc/create-reusable-components/01_overview.stories.mdx:24]:
```suggestion
- [Testing components](?path=/story/documentation-how-to-create-reusable-components-developing-components--page#testing-components)
```

**Review by SaiSan-WMDE — APPROVED** [review]

**Review by SaiSan-WMDE — APPROVED** [review]

**Comment by tzhelyazkova** [comment]
This PR can't be merged because the browser tests fail due to the branch name.
The commits have been cherry-picked and put into this new branch #156.

**Pull request closed without merging**